exercises/algorithms/README.md

```diff
@@ -81,35 +81,79 @@ return pIndex + 1
 
 ## :pushpin: Searching
 
-### Linear Search
```

Comment:
Are we no longer teaching Linear Search?

Reply:
We're still teaching it but removed the exercise related to that content. Instead of trying to implement four algorithms, students will implement one search and one sort algorithm.